ports/mimxrt10xx/supervisor/port.c

```diff
@@ -341,6 +337,52 @@ uint32_t port_get_saved_word(void) {
     return SNVS->LPGPR[1];
 }
 
+uint64_t port_get_raw_ticks(uint8_t* subticks) {
```

Comment:
Could you add a comment in all these `port.c' files that raw ticks are 1/1024s in each of those ports? It was more obvious the ticks were milliseconds in the old code because of `ticks_ms`, but now it's not so obvious.

Reply:
Never mind. I see it's in `supervisor/port.c`